router/router.go

```diff
@@ -1132,18 +1132,8 @@ func (*workerT) sendDestinationResponseToConfigBackend(payload json.RawMessage,
 	}
 }
 
-func durationBeforeNextAttempt(attempt int) (d time.Duration) {
-	b := backoff.NewExponentialBackOff()
-	b.InitialInterval = minRetryBackoff
-	b.MaxInterval = maxRetryBackoff
-	b.RandomizationFactor = 0
-	b.MaxElapsedTime = 0
-	b.Multiplier = 2
-	b.Reset()
-	for index := 0; index < attempt; index++ {
-		d = b.NextBackOff()
-	}
-	return
+func durationBeforeNextAttempt(attempt int) time.Duration {
+	return time.Duration(math.Min(float64(maxRetryBackoff), float64(minRetryBackoff)*math.Exp2(float64(attempt-1))))
```

Comment:
It is not clear if this can be called with attempt==0 or not, thus can we guard against such scenario & add a test scenario too?
```suggestion
	if attempt < 1 {
		attempt = 1
	}
	return time.Duration(math.Min(float64(maxRetryBackoff), float64(minRetryBackoff)*math.Exp2(float64(attempt-1))))
```